starter-code/index.js

```diff
@@ -1,10 +1,53 @@
-var SortedList = function() {};
+var SortedList = function(items, length) {
+  this.items = [];
+  this.length = this.items.length;
```

Comment:
they wanted `this.length = 0`, instead of having it hardcoded too `this.length = this.items.length`;